apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -0,0 +1,288 @@
+import { Injectable } from '@nestjs/common'
+import {
+  ConnectProfileStatus,
+  MentorshipMatchStatus,
+  RediLocation,
+  UserType,
+} from '@talent-connect/common-types'
+import * as AWS from 'aws-sdk'
+import * as jsforce from 'jsforce'
+import { ConMentorshipMatchesService } from '../con-mentorship-matches/con-mentorship-matches.service'
+import { ConProfilesService } from '../con-profiles/con-profiles.service'
+import { SfApiEmailTemplatesService } from '../salesforce-api/sf-api-email-templates.service'
+
+const sesConfig = {
```

Comment:
Can we move this into constructor and use `ConfigService` to get these values? (see another comment about how to do this)